Solutions/GoogleWorkspaceReports/Analytic Rules/GWorkspacePossibleMaldocFileNamesInGDRIVE.yaml

```diff
@@ -0,0 +1,33 @@
+id: d80d02a8-5da6-11ec-bf63-0242ac130002
+name: GWorkspace - Possible maldoc file name in Google drive
+description: |
+  'Detects possible maldoc file name in Google drive.'
+severity: Low
+requiredDataConnectors:
+  - connectorId: GoogleWorkspaceReportsAPI
+    dataTypes:
+      - GWorkspaceActivityReports
+queryFrequency: 1h
+queryPeriod: 1h
+triggerOperator: gt
+triggerThreshold: 0
+tactics:
+  - InitialAccess
+relevantTechniques:
+  - T1566
+query: |
+  GWorkspaceActivityReports
+  | where isnotempty(DocTitle)
+  | where DocTitle contains "invoice" or DocTitle contains "payment" or DocTitle contains "order" or DocTitle contains "fax" or DocTitle contains "scan" or DocTitle contains "transfer" or DocTitle contains "report" or DocTitle contains "bill"

```

Comment:
Instead of numerous 'contains' we can probably use the 'has_any' operator here. 
Eaxample use of 'has_any' in a query: https://github.com/Azure/Azure-Sentinel/blob/master/Hunting%20Queries/ASimProcess/imProcess_powershell_downloads.yaml

Reply:
These are not full terms (e.g. file names are like invoice2021-12-30.ext or CompanyXXXpayment.doc, etc.), that's why we use `contains` in query instead on `has_any` to cover more cases